### ISSUE TITLE
stop the backpropagation

### ISSUE THREAD

**Issue by guofuzheng** [keras-team/keras#11460]
![img_20181023_154623](https://user-images.githubusercontent.com/25239334/47344598-75f56b00-d6db-11e8-86a5-a81732ee33d0.jpg)
Hi, i have a network like this. And  i want to stop the backpropagation from CNN  to Inception. The Inception just get the gradients from LSTM. What should i do ? Thank you for your help.

**Comment by farizrahman4u** [comment]
```python
input = Input(...)
inception = ....
cnn = ...
lstm = ...
inception_out = inception(input)
cnn_input = Lambda(K.stop_gradient)(inception_out)
cnn_out = cnn(cnn_input)
lstm_out = lstm(inception_out)
```
